src/tasks/WasmAppBuilder/WasmLoadAssembliesAndReferences.cs

```diff
@@ -29,6 +29,7 @@ public class WasmLoadAssembliesAndReferences : Task
 
     private SortedDictionary<string, Assembly> _assemblies = new();
 
+    [RequiresAssemblyFiles(Message = "Using 'System.Reflection.Assembly.Location' will make ReferencedAssemblies to be empty")]
```

Comment:
This is a bit odd - this is an MSBuild task. What good is annotating it with `RequiresAssemblyFiles`? No upstream caller (i.e. MSBuild) is going to care about this attribute.

I assume we can just suppress the warning here - or maybe even turn off the `EnableSingleFileAnalyzer`, since this Task will only ever run in the context of MSBuild, and not a single file app.

Reply:
If the project will never run in single-file we should disable the analyzer for that project. Adding annotations for something which can never happen is just noise.